### ISSUE TITLE
Contents not navigable using keyboard only

### ISSUE THREAD

**Issue by CharlesBelov** [jsbin/jsbin#3429]
I am unable to navigate the site using keyboard only as the CSS specifies outline: none so I don't get any visual feedback as to where I am on the page.

Example page:  https://jsbin.com/wogadiyomu/edit?html,css,output

Press tab multiple times until I reach the link to Blog.

Expected result: Blog has an outline so I know that I am at the link to Blog.
Actual result: Blog has no outline so I have no way to know when I have reached it.

**Comment by CharlesBelov** [comment]
Closing as fixed
